### ISSUE TITLE
The form field cuts off the words

### ISSUE THREAD

**Issue by zawadZwd** [peiffer-innovations/json_dynamic_widget#109]
Howdy!

json_dynamic_widget: ^5.1.3+5
flutter: 3.3.9

In my case the form field is clearing off when I'm trying to type more than single word.
Here is how it looks like in the application :arrow_down:

![json_to_widget](https://user-images.githubusercontent.com/24791996/205298977-b9570da7-aa74-4729-8b0c-547a03fd3c86.gif)

And this is how I built json:

```json

{
  "type": "form",
  "child": {
    "type": "column",
    "children": [
      {
        "type": "list_tile",
        "args": {
          "title": {
            "type": "exclude_semantics",
            "child": {
              "type": "text_form_field",
              "id": "ID#ORDER_NUMBER",
              "args": {
                "decoration": {
                  "labelText": "Order number"
                },
                "validators": [
                  {
                    "type": "required"
                  }
                ]
              }
            }
          }
        }
      },
      {
        "type": "outlined_button",
        "id": "submit_button",
        "args": {
          "onPressed": "${validateForm('form_context')}"
        },
        "child": {
          "type": "save_context",
          "args": {
            "key": "form_context"
          },
          "child": {
            "type": "text",
            "args": {
              "text": "Submit"
            }
          }
        }
      }
    ]
  }
}
```

What did I miss? This is probably something obvious but I did not find similar issue, my apologies if this will be the duplicate.
Thank you for your help and time! :wink:

**Comment by crisperit** [comment]
I've confirmed that this issue occurs definitely on Android - but not on Web or Linux app

I'll take a look on that

**Comment by jpeiffer** [comment]
Found the issue.  The text box was both writing the updated value as well as subscribing to the events that trigger when the value is written.  That caused a feedback loop where only the last letter was stored.

This has been fixed with v5.1.6 that was deployed moments ago.  Thanks a ton for the report!
